WooCommerce/Classes/ViewRelated/Orders/Order Details/Product List Section/Product Details/AggregatedProductListViewController.swift

```diff
@@ -0,0 +1,148 @@
+import UIKit
+import Yosemite
+
+
+/// A simple Product List presented from the Order Details screen -> Shipping Labels Packages.
+///
+/// The list shows the product name and the quantity.
+///
+final class AggregatedProductListViewController: UIViewController {
+    @IBOutlet private var tableView: UITableView!
+
+    var viewModel: OrderDetailsViewModel!
+    private var products: [Product]? = []
+    var items: [AggregateOrderItem] = []
+
+    private let imageService: ImageService = ServiceLocator.imageService
```

Comment:
I really think we need to turn this into a normal VC with nib, to be able to inject these variables.

Reply:
I'm going to convert it into xib 👍 I did it just to conform to the existing view controllers that are in the `Orders` storyboard.